wdio.conf.js

```diff
@@ -231,7 +231,12 @@ exports.config = {
    */
   before: function (capabilities, specs) {
     assert = require('chai').assert;
-
+    browser.addCommand('getFlag', function () {
```

Comment:
I like this solution, and I'm glad that it works, but now I'm wondering why we're adding a command to the browser object that we only call once. Would it make sense to just load the object using an anonymous function? Maybe something like:

```suggestion
    global.flagObj = () => {
```

Reply:
It doesn't open URL this way. So to assign value to global variable this is how i have to call.